ambry-api/src/main/java/com.github.ambry/router/GetBlobOptions.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package com.github.ambry.router;
+
+public class GetBlobOptions {
+  private GetBlobRange range;
+
+  public GetBlobOptions withRange(GetBlobRange range) {
```

Comment:
I was thinking that if we add additional options in the future, we might not want to pass them all in through the constructor.